lasagne/layers/special.py

```diff
@@ -112,6 +113,67 @@ def get_output_for(self, input, **kwargs):
             return input
 
 
+class ScaleLayer(Layer):
+    """
+    lasagne.layers.ScaleLayer(incoming, scales=lasagne.init.Constant(1),
+    shared_axes='auto', **kwargs)
+
+    A layer that scales its inputs along certain axes.
```

Comment:
"scale along axes" confuses me -- what about just saying `A layer that scales its inputs by learned coefficients.`? Sharing coefficients over axes is explained fine in the `shared_axes` description.